### ISSUE TITLE
Windows Server 2016 package for PowerShell Core

### ISSUE THREAD

**Issue by pcgeek86** [PowerShell/PowerShell#2468]
There are Windows 10 and Windows 8.1 MSI installers for PowerShell Core. What is the recommended path for installing PowerShell Core on Windows Server 2016?

Cheers,
**Trevor Sullivan**
Docker Captain
Microsoft MVP: Cloud & Data Center Management
https://trevorsullivan.net
https://twitter.com/pcgeek86

**Comment by MaximoTrinidad** [comment]
Hi Trevor,

It would be nice to confirmed from them that the PowerShell core bits will work with Windows 10.  I did a leap of fate and installed it on my Windows Server 2016 TP5. It seems to work.

:)
Max

**Comment by pcgeek86** [comment]
@MaximoTrinidad Good to know -- what about the Windows Server 2016 GA bits that came out yesterday?

**Comment by MaximoTrinidad** [comment]
Trevor,
I'm setting up my Windows Server 2016 GA VM. I'll will post any issues here.
:)

**Comment by pcgeek86** [comment]
@MaximoTrinidad Cool, I guess we just need some official guidance at this point. I would not be surprised if it works, but renaming the packages for the next release might be necessary. @andschwa

**Comment by SteveL-MSFT** [comment]
Win10 and WS2016 are built off the same code base, so the Win10 pkg should work.  We should rename them to make this clear.

**Comment by joeyaiello** [comment]
I believe @raghushantha owns that piece. 

@raghushantha: can we get the next release packages renamed [a la the WMF 5.0 packages](https://www.microsoft.com/en-us/download/details.aspx?id=50395&751be11f-ede8-5a0c-058c-2ee190a24fa6=True)? Probably something like:

* powershell-6.0.0-alpha.10-win10-server2016-x64.zip/msi 
* powershell-6.0.0-alpha.10-win81-server2012r2-x64.zip

**Comment by MaximoTrinidad** [comment]
@joeyaiello I definitely agree in the proper renaming of the zip/msi packages.

@pcgeek86 , I got my Windows Server 2016 GA up and running with PowerShell Alpha10. The PowerShell Remoting part, I got Windows to Linux working. But, Linux to Windows not yet unless I'm still missing something. 

I did open an incident for the Linux to Windows Remoting: Remoting Linux to Windows fail after password #2473 

Also, I found out that Install-Packages Chocolatey won't work in either Windows PowerShell 5.1 or PowerShell Core. I had to use one of the command line example to make it work.

:)
